code/modules/mob/living/carbon/human/life.dm

```diff
@@ -899,7 +894,7 @@
 
 /mob/living/carbon/human/handle_vision()

```

Comment:
Is this even needed with how vision updates are handled now? same with species 'handle_vision`